service/src/main/java/bio/terra/workspace/service/crl/CrlService.java

```diff
@@ -136,10 +136,9 @@ public BigQueryCow createBigQueryCow(AuthenticatedUserRequest userRequest) {
    * Create a vanilla Bigquery client object, for testing things that aren't in CRL yet.
    * TODO(jaycarlton): PF-942 implement needed endpoints in CRL and use them here
    *
-   * @param userRequest
    * @return
    */
-  public Bigquery createNakedBigQueryClient(AuthenticatedUserRequest userRequest) {
+  public Bigquery createNakedBigQueryClient() {
```

Comment:
nit: To be consistent with naming and make it clear what credentials this client uses, perhaps rename this to `createWsmSaNakedBigQueryClient` or something similar? I think this method picks up the app default creds which should be the wsm sa.